src/extensions/extra/PDFView/thirdparty/pdfjs/web/viewer.css

```diff
@@ -1592,8 +1592,8 @@ html[dir='rtl'] .outlineItemToggler::before {
 /* TODO: file FF bug to support ::-moz-selection:window-inactive
    so we can override the opaque grey background when the window is inactive;
    see https://bugzilla.mozilla.org/show_bug.cgi?id=706209 */
-::selection { background: rgba(0,0,255,0.3); }
-::-moz-selection { background: rgba(0,0,255,0.3); }
+/*::selection { background: rgba(0,0,255,0.3); }
+::-moz-selection { background: rgba(0,0,255,0.3); }*/
```

Comment:
We can't make changes to this file, since it is an imported file from an upstream project.  If you need to override these, add `!important` to your rules in the `viewer-overrides.css` file above.